documents/customer.md

```diff
@@ -145,6 +145,248 @@ client.customer.fetch(customerId)
+| beneficiary_email  | string      | Email address of the beneficiary.  |
+| beneficiary_mobile | integer      | Mobile number of the beneficiary.  |
+| beneficiary_city  | string      | The name of the city of the beneficiary.  |
+| beneficiary_state | string      | The state of the beneficiary.  |
+| beneficiary_pin   | interger    | The pin code of the beneficiary's address.  |
+| ifsc_code          | string      | The IFSC code of the bank branch associated with the account.  |
+
+**Response:**
+```json
+{
+  "id" : "cust_1Aa00000000001",
+  "entity": "customer",
+  "name" : "Saurav Kumar",
+  "email" : "Saurav.kumar@example.com",
+  "contact" : "+919000000000",
+  "gstin":"29XAbbA4369J1PA",
+  "notes" : [],
+  "created_at ": 1234567890
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Delete Bank Account of Customer
+
+```py
+customerId = "cust_N5mywh91sXB69O"
+
+bankAccountId = "ba_N6aM8uo64IzxHu"
+
+client.customer.deleteBankAccount(customerId, bankaccountId)
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| customerId*  | string      | Unique identifier of the customer.  |
+| bankAccountId  | string      | The bank_id that needs to be deleted.  |
+
+**Response:**
+```json
+{
+    "id": "ba_Evg09Ll05SIPSD",
+    "ifsc": "ICIC0001207",
+    "bank_name": "ICICI Bank",
+    "name": "Test R4zorpay",
+    "account_number": "XXXXXXXXXXXXXXX0434",
+    "status": "deleted"
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Eligibility Check API
+
+```py
+client.customer.requestEligibilityCheck({
+    "inquiry": "affordability",
+    "amount": 500,
+    "currency": "INR",
+    "customer": {
+        "id": "elig_xxxxxxxxxxxxx",
+        "contact": "+919999999999",
+        "ip": "105.106.107.108",
+        "referrer": "https://merchansite.com/example/paybill",
+        "user_agent": "Mozilla/5.0"
+    }
+})
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| inquiry  | string      | List of methods or instruments on which eligibility check is required.  |
+| amount*  | string      | The amount for which the order was created, in currency subunits.  |
+| currency*  | string      | A three-letter ISO code for the currency in which you want to accept the payment.  |
+| customer*  | object      | All keys listed [here](https://razorpay.com/docs/payments/payment-gateway/affordability/eligibility-check/#eligibility-check-api)  |
+| instruments  | object | All keys listed [here](https://razorpay.com/docs/payments/payment-gateway/affordability/eligibility-check/#eligibility-check-api)  |
```

Comment:
The description does not seem right. Please add proper description. Kindly go through this file and make sure all text that is being written are proper.

Reply:
update description

 `Customer details.` for `customer*`  and
`Payment instruments on which an eligibility check is required.` for `instruments`

although it's showing `type` array in doc, but that is typo error it should be `object`

<img width="690" alt="Screenshot 2024-03-11 at 8 26 17 PM" src="https://github.com/razorpay/razorpay-python/assets/89454448/edecdcbe-5454-4344-8da3-e092df778ed3">